tests/PhpUnits/Helpers/HelperTest.php

```diff
@@ -855,8 +854,8 @@ public function testEncryptLinkSuccess() {
 			);
 
 			$this->assertEquals(
-				str_replace("&", "[amp;]", base64_encode("https://rawadymario.com/projects?page=2&category=2")),

```

Comment:
When replacing a string in the whole project, it's better to replace using Case Sensitive. Please change the domain to be dg-splash